### DISCUSSION TITLE
Modalprops for MediaPlayerCard

### DISCUSSION THREAD

**Discussion by alexberaut** [shannonhochkins/ha-component-kit#159] [Q&A] [answered]
Hi, I noticed that passing `hideLogbook` to `MediaPlayerCard`'s `modalProps` input wasn't hiding the logbook in the modal like I expected. This is how I'm using it

```
      <MediaPlayerCard
        entity={entityId}
        groupMembers={ALL_AREA_CONFIGS.flatMap(c => c.speakerGroupConfig.entityIds)}
        layout="slim"
        modalProps={{
          hideLogbook: true,
        }}
      />
```

I would expect to see `modalProps` passed through to the `MediaPlayerWrapper`'s `modalProps` input [here](https://github.com/shannonhochkins/ha-component-kit/blob/764265b66d58b256491175770d7d22cb44f68c4d/packages/components/src/Cards/MediaPlayerCard/index.tsx#L305) but am not seeing it, just the props referenced in `MediaPlayerControlsProps`. Passing any props to `modalProps` the way I have actually causes the modal not to show up at all. No console or type errors though. Am I missing something? Appreciate any help!

**Comment by shannonhochkins** [comment]
Interesting, I'll take a look at this as soon as I can for you

**Comment by shannonhochkins** [comment, marked as answer]
Just an FYI @alexberaut  this has been fixed with the latest release!

**Reply by alexberaut** [reply]
awesome thanks for the quick response!
